make_prg/from_msa/cluster_sequences.py

```diff
@@ -162,11 +171,22 @@ def merge_sequences(*seqlists: Sequences, first_seq: str) -> ClusteringResult:
                 first_seq_found = True
             else:
                 result.append(sequence)
-    if not first_seq_found:
-        raise ValueError(
-            "Provided sequence argument not found in provided list of sequences"
-        )
-    return ClusteringResult(None, [first_seq] + result)
+
+    # Note: this is an assert as is it the dev's responsibility to pass the correct first_seq argument (i.e. the user
+    # can't cause this error by erroneous input)
+    assert first_seq_found, f"Provided first sequence argument ({first_seq}) not found in provided list of sequences " \
+                            f"({seqlists})"
+
+    merged_unexpanded_sequences = [first_seq] + result
+    merged_expanded_sequences = SequenceExpander.get_expanded_sequences(merged_unexpanded_sequences)
```

Comment:
Not disagreeing with its use, but checking I understand: did you make a `SequenceExpander` class because you're expanding in different ways, eg from a list of sequences versus an MSA object?

It ends up being a bit verbose in my opinion, but I'm not really recommending anything else

Reply:
It was more for modularisation/refactoring purposes. In `seq_utils.py`, we have 5 variables and 4 functions related just with expanding sequences, so to make this clearer, I wrapped them all in a single `SequenceExpander` class. It is a sort of long source file, with 16 functions, so a bit of modularisation helps. I also thought of refactoring it into its single `sequence_expander.py` source file, but ended up leaving in `seq_utils.py` itself. You will be able to take a better look at it in the next PR, which will have utils-related code

Reply:
sure!